boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -244,4 +244,20 @@ TEST_F(DcmStateMachineTest, zero_torque_request_in_init_state)
         0.0f, App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface));
 }
 
+// DCM-16
+TEST_F(DcmStateMachineTest, zero_torque_request_in_fault_state)
+{
+    SetInitialState(App_GetFaultState());
+
+    // Start with a non-zero torque request to prevent false positive
+    App_CanTx_SetPeriodicSignal_TORQUE_REQUEST(can_tx_interface, 1.0f);
+    ASSERT_EQ(
+        1.0f, App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface));
```

Comment:
I don't think we need this `ASSERT_EQ` here, because we should already have tested this in the `CanTx` tests. Restricting asserts to just what we're focused on testing is actually a great way to help guide people as to the intent of the test.

Reply:
But if the default value for torque request is `0`, wouldn't the `CanTx` just check that `App_CanTx_Create` sets the torque request to `0`?   And since we want zero torque request in the fault state that would cause a false positive?

Reply:
Like if I remove this `ASSERT_EQ`, this test would pass even if I remove the code I added to `App_FaultState.c`

Reply:
Right, so we're checking the functionality of the `CanTx` here yeah? That should be a separate test.

Reply:
Ohhhhh I see and you mean now I thought you meant I should remove line 253 and 254. I gotcha now.

Reply:
Fixed!